lib/rails_charts/line_chart.rb

```diff
@@ -38,11 +38,15 @@ def generate_series_options
           type: type,
         }
       when Hash
-        data.map do |e| 
+        data.map do |e|
           {
             data: e[:data].is_a?(Hash) ? e[:data].values : e[:data],
             type: type,
-            name: e[:name]
+            name: e[:name],
+            yAxisIndex: e[:yAxisIndex]? e[:yAxisIndex] : 0,
+            stack: e[:stack].presence || nil,
+            areaStyle: e[:areaStyle] == {} ? {} : nil,
+            emphasis: e[:emphasis].presence || nil,
```

Comment:
Is this something where we can just iterate through `e` and dynamically construct the symbol & value rather than having to manually define each?  Falling back to nil by default, and could case out stuff like `yAxisIndex1 for different behaviour?